src/node_modules/app/components/AccessibilityMenu.svelte

```diff
@@ -65,23 +80,38 @@
 		<label for=''>{label}</label>
 		{#if $_currentSetting.values}
 			<div class='controlContainer'>
-				{#each $_currentSetting.values as value, index}
-					<div>
-						<label for='input-{index}'>
-							{#if showLabels}
+				{#if useRadios}
+					{#each $_currentSetting.values as value, index}
+						<div>
+							<label for='input-{index}'>
 								{$_formatValue(value)}
-							{/if}
-						</label>
-						<input
-							name='input-{index}'
-							checked={$_currentSetting.value === value}
-							class='clickable'
-							on:change={() => updateCurrentValue(value)}
-							type='radio'
-							{value}
-						>
-					</div>
-				{/each}
+							</label>
+							<input
+								name='input-{index}'
+								checked={$_currentSetting.value === value}
+								class='clickable'
+								on:change={() => updateCurrentValue(value)}
+								type='radio'
+								{value}
+							>
+						</div>
+					{/each}
+				{:else}
+					<button
```

Comment:
We need to add a specific class to these buttons because right now they're styled like the side buttons:

<img width="1224" alt="Screenshot 2021-05-26 at 09 17 52" src="https://user-images.githubusercontent.com/1309648/119626981-a9548500-be03-11eb-9d5a-c33abadaf5fc.png">

We need to:
- make these button 30px of size
- style the disabled state in such a way that both the icon and the border are coloured with `var(--color-grey-180)`